radixdlt-java-common/src/main/java/com/radixdlt/utils/Bytes.java

```diff
@@ -281,4 +282,19 @@ private static byte fromHexNybble(char value) {
     }
     return ret;
   }
+
+  /** Checks whether a given array consists of only zero bytes. */
+  public static boolean allZeros(byte[] arr) {
```

Comment:
I like functions returning booleans to be clearly questions in their names - eg start with `is` or `should` etc, to distinguish them from functions returning something (which are nouns), so eg I'd prefer `isAllZeros` here